src/components/Contact.jsx

```
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
```

ESLint

[error] 'useState' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)